lua/lspconfig/server_configurations/clangd.lua

```diff
@@ -21,6 +21,34 @@ local function switch_source_header(bufnr)
   end
 end
 
+local function symbol_info(bufnr)
+  bufnr = util.validate_bufnr(bufnr)
+  local clangd_client = util.get_active_client_by_name(bufnr, 'clangd')
+  if not clangd_client then
```

Comment:
```suggestion
  if not clangd_client or not client:support_method('textDocument/symbolInfo') then
```

Reply:
I don't really understand what you suggest here, what is `client` and is there a method called `support_method`?

Reply:
sry a typo here `clang_client.supports_method`